### ISSUE TITLE
MAKE DELETION OF ENTITY NOT CHANGE LOCATIONS OF ALL OTHER ENTITIES.

### ISSUE THREAD

**Issue by lucianojamimbo** [lucianojamimbo/schplungeon#2]
```
def delete(IDtoberemoved):
    del entIDlist[IDtoberemoved]
    del entitylocationlist[IDtoberemoved]
    del entstat[IDtoberemoved]
```

this is terrible

**Comment by lucianojamimbo** [comment]
potential solution: use a for loop to subtract 1 from each item in entIDlist above IDtoberemoved

**Comment by lucianojamimbo** [comment]
fixed in commit 666300edaaf62d93683afb1adb0f3d29b6ab0c49

```
def delete(IDtoberemoved):
    global entIDlist
    global entitylocationlist
    global entstat
    newidlist = []
    for item in entIDlist[:IDtoberemoved]:
        newidlist.append(item)
    for item in entIDlist[IDtoberemoved:]:
        newidlist.append(item-1)
    entIDlist = newidlist[:]
    del entIDlist[IDtoberemoved]
    del entitylocationlist[IDtoberemoved]
    del entstat[IDtoberemoved]
```
